src/pages/manage/backup-restore.tsx

```diff
@@ -0,0 +1,99 @@
+import { Box, Button, notificationService } from "@hope-ui/solid";
+import {r} from "~/utils";
+
+const BackupRestore = () => {
+  return (
+    <Box>
+      <Button
+        onClick={async () => {
+          await backup();
+        }}
+      >
+        Backup
+      </Button>
+      <Button
+        onClick={async () => {
+          await restore();
+        }}
+      >
+        Restore
+      </Button>
+      </Box>
+  )
+        
+};
+
+function download(filename: string, data: any) {
+  const blob = new Blob([JSON.stringify(data,null,2)], {
+    type: "application/json",
+  });
+  const url = URL.createObjectURL(blob);
+  const a = document.createElement("a");
+  a.href = url;
+  a.download = filename;
+  a.click();
+  URL.revokeObjectURL(url);
+}
+
+const backup = async() => {
+  const date = new Date();
+  const year = date.getFullYear();
+  const month = date.getMonth() + 1;
+  const dates = date.getDate();
+
+  const todaysdate = year + "-" + month + "-" + dates;
+  const data = r.get("/admin/setting/list");
+  if ((await data).code == 200)
+  {
+    download("alist_backup_" + todaysdate + ".json", (await data).data);
```

Comment:
For all request with token, we need use `useFetch` and `handleResp` to fetch and handle.

Reply:
> For all request with token, we need use `useFetch` and `handleResp` to fetch and handle.

Well, I got some issues while using these things, Json.stringify will lose all the data like this 

![image](https://user-images.githubusercontent.com/58730749/201506812-14616995-e4ca-4deb-8e85-22121e079341.png)

![image](https://user-images.githubusercontent.com/58730749/201506823-38ce397d-6244-412a-97e9-caddf9d069fb.png)

![image](https://user-images.githubusercontent.com/58730749/201506840-3fb18c52-804c-43a4-85e2-099827764bf2.png)

Reply:
I've got no idea how to fix it, could you please fix this issue?
https://github.com/alist-org/alist-web/commit/42fc1df99fa776ce8c44201d85e0501931145b3e